target/linux/mediatek/mt7622/base-files/etc/board.d/02_network

```diff
@@ -29,9 +30,28 @@ mediatek_setup_interfaces()
 	esac
 }
 
+mediatek_setup_macs()
+{
+	local board="$1"
+	local lan_mac=""
+	local wan_mac=""
+	local label_mac=""
+
+	case $board in
+	ruijie,rg-ew3200gx-pro)
+		lan_mac=$(macaddr_add $(mtd_get_mac_ascii u-boot-env ethaddr) 1)
+		;;
+	esac
+
+	[ -n "$lan_mac" ] && ucidef_set_interface_macaddr "lan" $lan_mac
+	[ -n "$wan_mac" ] && ucidef_set_interface_macaddr "wan" $wan_mac
```

Comment:
This line is now unused and can be removed. Once there is a board which needs it, we can easily add it again.

Reply:
✅Removed.